clients/admin-ui/src/features/datastore-connections/ConnectionTypeLogo.tsx

```diff
@@ -48,7 +48,7 @@ const ConnectionTypeLogo: React.FC<ConnectionTypeLogoProps & ImageProps> = ({
   };
   const getAltValue = (): string => {
     if (isDatastoreConnection(data)) {
-      return data.name;
+      return data.name ?? data.key;
```

Comment:
I'm guessing this key is autogenerated? might not be a great alt text, but I'm not sure what would be better...

Reply:
key is used in other places. It should be okay 👍